packages/grant-explorer/src/features/discovery/RoundCard.tsx

```diff
@@ -1,6 +1,6 @@
 import { ChainId, renderToPlainText, truncateDescription } from "common";
 import { RoundOverview } from "../api/rounds";
-import { getDaysLeft } from "../api/utils";
+import { getDaysLeft, payoutTokens } from "../api/utils";
```

Comment:
For the future, exporting getters > exporting data: https://github.com/gitcoinco/grants-stack/issues/2158

Reply:
I'm keeping this in mind, that will be part of a larger refactor of all static data across apps like tokens, networks, assets etc.